Makefile

```diff
@@ -288,6 +288,10 @@ bin/releases/git-lfs-%-$(VERSION).zip : $(RELEASE_INCLUDES) bin/git-lfs-%.exe
 bin/releases/git-lfs-$(VERSION).tar.gz :
 	git archive -o $@ --prefix=git-lfs-$(patsubst v%,%,$(VERSION))/ --format tar.gz $(VERSION)
 
+.PHONY : release-linux
```

Comment:
I think that it's unnecessary, but it may be good to document this target in a similar fashion as `release-windows` below.